docs/README.md

```diff
@@ -55,51 +36,75 @@ There are several optional parameters that could be passed into `CreateVolumeReq
+|v0.1.0                     |amazon/aws-ebs-csi-driver:0.1.0-alpha|
 
-2. The driver can use the EC2 instance roles, otherwise add AWS credentials of the IAM user to the [deploy/kubernetes/secret.yaml](../deploy/kubernetes/secret.yaml) file.
+## Features
+* **Static Provisioning** - create a new or migrating existing EBS volumes, then create persistence volume (PV) from the EBS volume and consume the PV from container using persistence volume claim (PVC).
+* **Dynamic Provisioning** - uses persistence volume claim (PVC) to request the Kuberenetes to create the EBS volume on behalf of user and consumes the volume from inside container.
+* **Mount Option** - mount options could be specified in persistence volume (PV) to define how the volume should be mounted.
+* **Block Volume** - consumes the EBS volume as a raw block device for latency sensitive application eg. MySql
+* **Volume Snapshot** - creating volume snapshots and restore volume from snapshot.
+* **NVMe** - consume NVMe EBS volume from EC2 [Nitro instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances).
 
+## Prerequisites
+* Get yourself familiar with how to setup Kubernetes on AWS
+* If you are managing EBS volumes using static provisioning, get yourself familiar with [EBS volume](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html).
+* Have a working Kubernetes cluster:
+  * Enable flag `--allow-privileged=true` for `kubelet` and `kube-apiserver`
+  * Enable `kube-apiserver` feature gates `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true`
+  * Enable `kubelet` feature gates `--feature-gates=CSINodeInfo=true,CSIDriverRegistry=true,CSIBlockVolume=true`
+
+## Installation
+Checkout the project:
+```sh
+git clone https://github.com/kubernetes-sigs/aws-ebs-csi-driver.git
+cd aws-ebs-csi-driver
 ```
-apiVersion: v1
-kind: Secret
-metadata:
-  name: aws-secret
-stringData:
-  key_id: [aws_access_key_id]
-  access_key: [aws_secret_access_key]
-```
-
-3. Apply the secret using `kubectl apply -f deploy/kubernetes/secret.yaml` if required.
 
-4. Grant only required permissions to the CSI driver. Use this sample [IAM policy](example-iam-policy.json) and add it to the worker nodes in the cluster.
-
-5. Deploy the csi-provisioner, csi-attacher and csi-node manifests to the cluster in one step:
+Edit the [secret manifest](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/deploy/kubernetes/secret.yaml) using your favorite text editor. The secret should have [enough IAM permission](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/example-iam-policy.json) to create EBS volume. Then deploy the secret:
+```sh
+kubectl apply -f deploy/kubernetes/secret.yaml
+```
 
+Install the `CSINodeInfo` CRD on the cluster:
+```sh
+kubectl create -f https://raw.githubusercontent.com/kubernetes/csi-api/release-1.13/pkg/crd/manifests/csinodeinfo.yaml
```

Comment:
This shouldn't be necessary anymore starting from 1.14, as these CRDs became built-in objects.

Reply:
Since people might use it for version < 1.14, updated to only do this if this is the case.